src/components/TimeseriesDataExport/interfaces.ts

```diff
@@ -64,10 +64,6 @@ export interface TimeseriesDataExportProps {
    * Flag that shows/hides modal with form
    */
   modalWidth?: number;
-  /**
-   * Limit of cells for generated CSV documents, can't be greater then 10000
-   */
-  cellLimit?: number;
```

Comment:
we cannot just remove the interface, but since it's not needed anymore, we could mention that it's "Deprecated. This property will be ignored if passed". or something like that

Reply:
Agreed!